packages/twenty-server/src/engine/api/graphql/graphql-query-runner/utils/wrapper-with-double-quote-when-uppercase.ts

```diff
@@ -0,0 +1,5 @@
+export const wrapperWithDoubleQuoteWhenUpperCase = (value: string) => {
```

Comment:
Why don't we always add quotes? I know we don't add much comments in the codebase but maybe it's worth adding one here because it's not clear to me which we cannot always add quotes and get rid of this function entirely